plugins/aea-crypto-ethereum/tests/test_ethereum.py

```diff
@@ -0,0 +1,305 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
```

Comment:
Why does this now show relative changes?

Reply:
because we have another file in `tests/test_crypto/test_ethereum.py`.

Reply:
What's in the other one? Why is not everything here?

Reply:
a test on an ethereum contract